packages/candid/src/idl.test.ts

```diff
@@ -152,6 +152,20 @@ test('IDL encoding (tuple)', () => {
   );
 });
 
+test('IDL encoding (arraybuffer)', () => {
+  // ArrayBuffer, encode only.
+  testEncode(
```

Comment:
I didn't change the decoding behavior because it will be a breaking change.

But I will suggest decoding should be changed too. What do people think?

Reply:
Commit [a1e97d6](https://github.com/dfinity/agent-js/pull/566/commits/a1e97d646d00c26f15aa1f78d63559cfeb393775) adds a fast decoding path too. This changes the return type from Array to ArrayBuffers, and will likely break user code when they upgrade.